app/src/ActionModal/components/DelegationModal.vue

```diff
@@ -325,6 +325,12 @@ export default {
     enhancedTargetValidator() {
       return validatorEntry(this.targetValidator)
     },
+    isPolkadotController() {
```

Comment:
I'd rather call it `isPolkadotMinAmount` or something in this fashion

Reply:
Oh I disagree, I think it describe its function better.